internal/rules/prefer_optional_chain/schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "prefer_optional_chain_options": {
+      "type": "object",
+      "properties": {
+        "allowPotentiallyUnsafeFixesThatModifyTheReturnTypeIKnowWhatImDoing": {
```

Comment:
Probably should just skip adding this option, in light of https://github.com/oxc-project/tsgolint/issues/425